### ISSUE TITLE
is the def of male germ cell correct?

### ISSUE THREAD

**Issue by ukemi** [obophenotype/cell-ontology#574]
male germ cell: germ cell and (capable of some spermatogenesis). Should this be capable of part of some spermatogenesis. The individual precursors like spermatogonium are not entirely capable of carrying out the process.

**Comment by paolaroncaglia** [comment]
@dosumis @shawntanzk could you please confirm the correct object property to use for this case? Both these guidelines say "capable of some": 
https://obophenotype.github.io/cell-ontology/patterns/cellCapableOfBiologicalProcess/
https://github.com/obophenotype/cell-ontology/blob/draft_relations_doc/documentation/relations_guide.md

**Comment by paolaroncaglia** [comment]
@shawntanzk yeah, that's partly why I'm confused. The broad parent term could be capable of the whole process because the _sum_ of its children would be? 
Let's tag @ukemi who raised the ticket (hi David!).

**Comment by ukemi** [comment]
Hi Paola!

Old ticket. I think the reason I raised the issue was two-fold. First, does it really mean the sum of all of the subclasses or does it mean any instance of the class is capable of the whole process? If it is the latter, then no one individual of the subtype can carry out the process by itself.  This assumes that as we go through successive stages of spermatogenesis cells divide and change identity, suggested by the children of male germ cell. I also had the concern that if I recall correctly in some organisms spermatogenesis also requires Sertoli cells to contribute to the process. If that is the case, then the definition wouldn't be necessary and sufficient either way. Does this make sense? I don't think spermatogenesis is just the male germ cell(s) going through the developmental process but the process itself, which requires more.

**Comment by shawntanzk** [comment]
Oh sorry, should have probably checked CL before responding >.< didn't realise spermatogonium was a subclass of male germ cell. From the definition of `spermatid`,`spermatocyte`, and `spermatogonium`, I think the problem is that they are wrongly placed as a subclass of male germ cell - those cells are not male germ cells but rather develops from male germ cells. I think we should instead categorise them as precursor cells (that's the right superclass right?), and have a `develop from some male germ cell` axiom on them instead

**Comment by shawntanzk** [comment]
Thanks @ukemi, sorry for confusion, been a long day and my brains not quite on it today >.< 

>  does it really mean the sum of all of the subclasses or does it mean any instance of the class is capable of the whole process?

I think because all subclasses inherit the axiom, it means means any instance of the class is capable of the whole process

> if I recall correctly in some organisms spermatogenesis also requires Sertoli cells to contribute to the process

If this is true and is in the GO process, then I think you are right that it should instead be `capable of part of`. 

Overall I think you are right that it should be changed. I don't think it should be an equivalent class though for the above reason - esp if something like Sertoli cell is involved.

**Comment by ukemi** [comment]
Yeah, it gets to the murky subject of process boundaries, but I think that the things that the Sertoli cells do would be considered part of the process of spermatogenesis. PMID:29462262

**Comment by paolaroncaglia** [comment]
@ukemi @shawntanzk @dosumis 
Thank you for confirming. I edited 'male germ cell' as follows:

- added PMID:29462262
- changed def from "Male germ cell is a germ cell that supports male gamete production." to "A germ cell that supports male gamete production. In some species, non-germ cells known as Sertoli cells also play a role in spermatogenesis."
- replaced the equivalence with subclasses 'germ cell' + 'capable of part of' some spermatogenesis
- added a seeAlso pointing to this ticket.

Closing now, but feel free to reopen if you have any concern.
